packages/core/ios/Tests/DdSdkTests.swift

```diff
@@ -449,7 +449,42 @@ class DdSdkTests: XCTestCase {
         let ddContext = try XCTUnwrap(CoreRegistry.default as? DatadogCore).contextProvider.read()
         let userInfo = try XCTUnwrap(ddContext.userInfo)
 
-        XCTAssertEqual(userInfo.id, "abc-123")
+        XCTAssertEqual(userInfo.id, "id_123")
+        XCTAssertEqual(userInfo.name, "John Doe")
+        XCTAssertEqual(userInfo.email, "john@doe.com")
+        XCTAssertEqual(userInfo.extraInfo["extra-info-1"] as? Int64, 123)
+        XCTAssertEqual(userInfo.extraInfo["extra-info-2"] as? String, "abc")
+        XCTAssertEqual(userInfo.extraInfo["extra-info-3"] as? Bool, true)
+    }
+    
+    func testSettingUserInfoOptionalId() throws {
```

Comment:
I added this extra test so we test both cases, with and without user id. On Android there are already specific tests for each scenario.